docs/blocks/administration/Pricing.mdx

```diff
@@ -0,0 +1,892 @@
+import { Meta, Unstyled } from '@storybook/addon-docs';
+import { Pricing } from './Pricing';
+import { PricingFull } from './PricingFull';
+import { PricingMinimal } from './PricingMinimal';
+import { Tabs, TabPanel, TabList, Tab } from '../../../src/layout/Tabs';
+
+<Meta title="Blocks/Administration/Pricing" />
+
+# Pricing
+
+<Unstyled>
+  <Tabs direction="rtl">
+    <TabList>
+      <Tab>
+        Demo
+      </Tab>
+      <Tab>
+        Code
+      </Tab>
+    </TabList>
+    <TabPanel>
+      <div className="light:bg-white light:text-black flex flex-col border justify-center items-center border-gray-500 p-7">
+        <Pricing />
+      </div>
+    </TabPanel>
+    <TabPanel>
+  ```tsx
+  import { useState } from 'react';
+import {
+  Card,
+  Divider,
+  List,
+  ListItem,
+  MotionGroup,
+  MotionItem,
+  Stack
+} from '../../../src/layout';
+import { Button } from '../../../src/elements/Button';
+import { cn } from '../../../src/utils';
+import { Chip } from '../../../src/elements';
```

Comment:
i wonder if when we put the "code" part in, we should just have this import from Reablocks instead of the relative paths